Detectors/EMCAL/base/src/EMCGeometry.cxx

```diff
@@ -511,6 +506,8 @@ void EMCGeometry::Init(const Text_t* mcname, const Text_t* mctitle)
 
 void EMCGeometry::PrintStream(std::ostream& stream) const
 {
+  std::function<bool(const std::string_view, const char *)> findSubstring = [](const std::string_view tocheck, const char *teststring) { return tocheck.find(teststring) != std::string::npos; };
```

Comment:
repetition of same function somewhere else? You could consider implementing this lambda in the scope of the file (possibly in an anonymous namespace to prevent usage from outside).